mlflow/tracking/client.py

```diff
@@ -1421,6 +2229,39 @@ def search_model_versions(self, filter_string):
     def get_model_version_stages(self, name, version):  # pylint: disable=unused-argument
         """
         :return: A list of valid stages.
+
+        .. code-block:: python
+            :caption: Example
+
+            import mlflow.sklearn
+            from mlflow.tracking import MlflowClient
+            from sklearn.ensemble import RandomForestRegressor
+
+            mlflow.set_tracking_uri("sqlite:///mlruns.db")
+            params = {"n_estimators": 3, "random_state": 42}
+            name = "RandomForestRegression"
+            rfr = RandomForestRegressor(**params)
+
+            # Log MLflow entities
+            with mlflow.start_run() as run:
+                mlflow.log_params(params)
+                mlflow.sklearn.log_model(rfr, artifact_path="models/sklearn-model")
+
+            # Register model name in the model registry
+            client = MlflowClient()
+            client.create_registered_model(name)
+
+            # Create a new version of the rfr model under the registered model name
+            # fetch valid stages
+            model_uri = "runs:/{}/models/sklearn-model".format(run.info.run_id)
+            mv = client.create_model_version(name, model_uri, run.info.run_id)
+            stages = client.get_model_version_stages(name, int(mv.version))
```

Comment:
Would this line raise an error if we removed `int` and passed `mv.version` as a string to `get_model_version_stages `?

Reply:
Yes, it expects an int.

Reply:
Let me try it.

Reply:
It seems like get_model_version(), update_model_version(), and transition_model_version_stage() can take mv.version, with explict cast. Will change it.